### PR TITLE
Profile Page UI

### PR DESCRIPTION
**Pull request by shiara02**
### OVERVIEW

Created User Interface for the Profile Page

### NOTES

none

### SNIPPETS

image
image

### CODE REVIEW

File: src/resources/views/layout/navbar.blade.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        </div>
+        
+        <div class="ml-auto">
+            <img src="profile.jpg" alt="Profile Picture" width="40" height="40" class="rounded-circle">
+            <span class="text-white mr-2">User Name</span>
+        </div>
+    </nav>
+
+    @yield('content')
+
```
Comment (bpocrafael) [inline src/resources/views/layout/navbar.blade.php]: trailing white space.

**Review by bpocrafael — DISMISSED** [review]

File: src/resources/views/layout/navbar.blade.php (hunk 1 above)
Comment (shiara02) [reply src/resources/views/layout/navbar.blade.php]: I resolved this comment, sir. Thank you!

**Review by hipe-francis — COMMENTED** [review]
Create a new PR for this after ma-merge mga fixes nyo. thanks

File: src/resources/views/layout/navbar.blade.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                    <a class="nav-link" href="#">Settings</a>
+                </li>
+            </ul>
+        </div>
+        
+        <div class="ml-auto">
+            <img src="profile.jpg" alt="Profile Picture" width="40" height="40" class="rounded-circle">
+            <span class="text-white mr-2">User Name</span>
+        </div>
+    </nav>
```
Comment (hipe01mark) [inline src/resources/views/layout/navbar.blade.php:34]: Create separate file.

**Pull request closed without merging**